docs/sources/installation/mac.md

```diff
@@ -10,37 +10,61 @@ weight = 500
 
 # Install on macOS
 
-This page provides instructions to help you install Grafana on macOS. 
+This page explains how to install Grafana and get the service running on your macOS. It has the following topics:  
```

Comment:
We don't generally use the mini-TOC. It's redundant with the auto-generated side menu. Please remove.